frontend/src/pages/Report.tsx

```diff
@@ -422,6 +425,11 @@ export const Report = () => {
           })}
       </section>
       <section className="save-button-container">
+        {showUnsavedMessage && (
+          <div class="unsaved-alert-p">
+            <p>⚠ You have unsaved changes</p>
```

Comment:
If you use an icon and an img tag instead, it becomes more semantic html and accessible because you can add an alt tag. Here's a nice icon: https://icons.getbootstrap.com/icons/exclamation-triangle/ 
You can download it, add it to the icons folder and insert it with something like this
```suggestion
            <img src={...} alt="warning triangle" className="..." />
```

Reply:
You have unsaved changes is self-explanatory, I think.